main.cpp

```diff
@@ -80,7 +81,7 @@ void main_window::on_command(int id)
 			if (GetOpenFileName(&ofn))
 			{
 				imagePath = Image::FromFile(path);
```

Comment:
A što ako je već otvorena slika, niste obrisali prethodni objekt (prijedlog: koristite `std::unique_ptr`)?